package.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "EverydayHero <edh-dev@everydayhero.com>",
   "name": "edh-widgets",
-  "version": "0.2.5",
+  "version": "0.2.6",
```

Comment:
Can we bump the revision in a separate PR please.  There's a couple of things to be released.